idd/Energy+.idd.in

```diff
@@ -18632,10 +18632,15 @@ SurfaceProperty:LocalEnvironment,
        \type object-list
        \object-list SurroundingSurfacesNames
        \note Enter the name of a SurfaceProperty:SurroundingSurfaces object
-   A5; \field Outdoor Air Node Name
+   A5, \field Outdoor Air Node Name
        \type object-list
        \object-list OutdoorAirNodeNames
        \note Enter the name of an OutdoorAir:Node object
+   A6; \field Ground Surfaces Object Name
+       \type object-list
+       \object-list GroundSurfacesNames
+       \note Enter the name of a SurfaceProperty:GroundSurfaces object
+	   \note This field is used with SurfaceProperty:GroundSurfaces object
```

Comment:
This text needs correction. ~\note This field is used with SurfaceProperty:GroundSurfaces object~.